### ISSUE TITLE
Make Dropdown attribute Multi-select capable

### ISSUE THREAD

**Issue by objecttothis** [opensourcepos/opensourcepos#2663]
- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.1
- OSPOS git commit hash is: 
- PHP version is: 
- MySQL or MariaDB version is: 
- OS and version is: 
- WebServer is: 
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### New Feature

Currently the DROPDOWN attribute type is standard single-select dropdown, but in some cases a user may want to multi-select values.  For example, if you have an attribute called Subtitle Languages, you may want to select English, French and German in the dropdown since all three are included in that particular title.

**Comment by daN4cat** [comment]
Yep, good point. Although I opted not to use attributes but retain my additional fields and business logic around it.

**Comment by objecttothis** [comment]
> 
> 
> Yep, good point. Although I opted not to use attributes but retain my additional fields and business logic around it.

Out of curiosity, are you just punting until later, or dropping the idea of migrating all together?  Not that it really matters, but I was curious.
